### PR TITLE
integration test cleanup

### PR DESCRIPTION
**Pull request by claasd**
integration tests now block each other. Only the newest integration test is executed

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,4 +1,19 @@
 stage("Unit Tests") {
+    unitTests()
+}
+
+stage("IntegrationTests") {
+    String folderName = env.BRANCH_NAME.replace('/', "-")
+    // make sure only one build is executing integration tests.
+    // abort older builds if multiple builds are waiting
```
Comment (CodingSpiderFox) [inline Jenkinsfile]: I'm not sure whether it is good to abort waiting builds. 

Don't we lose the possibility to see which commit broke something then?

File: Jenkinsfile [hunk 2]
```diff
@@ -61,6 +68,7 @@ void setupAndRunIntegrationTest(String folderName, String testFile) {
     String fullPath = "/cab/buildTools/bop-infra_tmp/${folderName}/${testFile}"
     jobDsl scriptText: """
     pipelineJob("${fullPath}") {
+        quietPeriod(0)
```
Comment (CodingSpiderFox) [inline Jenkinsfile]: Why is this needed? Does it override a default value?

Comment (claasd) [inline Jenkinsfile]: Yes, the default is five seconds

File: Jenkinsfile (hunk 1 above)
Comment (claasd) [inline Jenkinsfile]: Unit tests are executed for every push. integration tests are only aborted if multiple commits (and pushes) are done within several minutes.

Comment (CodingSpiderFox) [inline Jenkinsfile]: Ok, we will see how often we run into this case.
